### DISCUSSION TITLE
Possible to process more than one person?

### DISCUSSION THREAD

**Discussion by mjhmilla** [freemocap/freemocap#264] [Q&A] [answered]
I recently used freemocap to record the kinematics of two people playing table tennis. Freemocap worked wonderfully on one of the players, but the other was ignored. Is there any way to have more than one person analyzed?

In the case of table tennis, I can imagine doing the following: make two copies of the video, in one video block out the part of the image that contains player A and analyzed player B, in the second video block out player B and analyze player A. Is there a more elegant way to do this?

Thanks again for the wonderful work on freemocap!

mjhmilla

**Comment by aaroncherian** [comment, marked as answer]
Hey! 

So at the moment - we don't have a built in way to do multi-person analysis, but it is in the plans. 

There are different levels to how one can go about it - and the one you described is the most brute force method but it definitely works (and I've done it before myself). Especially in the case of table tennis, if either player A or player B aren't moving around too much in the frame, you could just draw a black box around the area where they are in a video editor (I've used Adobe Premiere to do this) - and then you'd essentially have two sets of videos to run through FreeMoCap.

The longer term goal is to implement a method to draw a bounding box around specific people, run the pose estimation software separately on the areas defined by the boxes, and have an automated way to make a reconstruction of multiple people. It's definitely on the to-do list.
